### DISCUSSION TITLE
Gitlab CI config

### DISCUSSION THREAD

**Discussion by niugm** [napi-rs/napi-rs#406] [Q&A] [answered]
Is there a recommended gitlab configuration？

**Comment by Brooooooklyn** [comment, marked as answer]
Not yet, but gitlab configuration should be simpler than github config. I will provide a example project in `gitlab.com` when I had time.

**Reply by niugm** [reply]
That's great,waiting for your good news

**Comment by lpsm-dev** [comment]
I don't think so, but you can look at their default [Auto-DevOps files](https://gitlab.com/gitlab-org/gitlab/-/tree/master/lib/gitlab/ci/templates)

**Comment by 8de2fdb0** [comment]
@Brooooooklyn I guess you never had time
